Jumpscale/builder/apps/BuilderCoreX.py

```diff
@@ -115,28 +115,24 @@ def running(self):
     def stop(self):
         # killing the daemon
         pane = j.tools.tmux.pane_get(self.NAME)
-        processes = pane.process_obj.children(True)
-        for process in processes:
-            process.kill()
         pane.kill()
-        """ j.sal.process.killProcessByName(self.NAME) """
 
     @builder_method()
     def test(self):
         if self.running():
             self.stop()
 
         self.start()
-        cc = j.clients.corex.get("localhost", 7681)
+        cc = j.clients.corex.get(addr="localhost", port=7681)
         assert cc.process_list() == []
 
-        cc.process_start("/bin/true")
+        cc.process_start("true", "/bin/true")
 
-        cc = j.clients.corex.get("localhost", 7681)
+        cc = j.clients.corex.get(addr="localhost", port=7681)
         assert len(cc.process_list()) == 1
 
         self.stop()
-
+        assert self.running() == False
```

Comment:
asserting `self.running() is False` is a stronger assertion, checking on both value and type